pennylane/ops/op_math/sprod.py

```diff
@@ -166,13 +166,13 @@ def label(self, decimals=None, base_label=None, cache=None):
     @property
     def data(self):
         """The trainable parameters"""
-        return [[self.scalar], self.base.data]  # Not sure if this is the best way to deal with this
+        return [self.scalar] + self.base.data
```

Comment:
```suggestion
        return [self.scalar, *self.base.data]
```
This seems to be a little faster.